cds-snc/cppd-medical-report-sails
# Enable error styling in macros

Follow up to #4 

## Adds
- Enables styling of individual inputs on validation error, ie:
  ![image](https://user-images.githubusercontent.com/1187115/71025438-0ad3d880-20d5-11ea-8bb8-6b5c7086c34b.png)

  The new validations format allows for multiple validation errors per field. Previously, the above 
  example would only show one of the two errors at a time.
- enable [JinjaCompatibility mode for Nunjucks](http://mozilla.github.io/nunjucks/api.html#installjinjacompat)


## Fixes
- Setting focus on firstError

## To test
- navigate to /en/personal
- submit form without entering anything
- notice the error state styling on individual form elements
- also, the cursor should be set to the first error field (only applies to input and textareas)
